### PR TITLE
Correcting sign of the normal

### PR DESCRIPTION
**Pull request by loumalouomega**
Basically that

### CODE REVIEW

**Comment by loumalouomega** [comment]
For some reason when I commited #646 I dind't have that problem with the normal

File: applications/StructuralMechanicsApplication/custom_conditions/line_load_condition_2d.cpp [hunk 1]
```diff
@@ -159,12 +159,12 @@ namespace Kratos
...
         
         for ( unsigned int point_number = 0; point_number < integration_points.size(); point_number++ )
-        {            
+        {                   
             const double det_j = GetGeometry().DeterminantOfJacobian( integration_points[point_number] );
 
             const double integration_weight = GetIntegrationWeight(integration_points, point_number, det_j); 
             
-            const array_1d<double, 3> normal = GetGeometry().Normal( integration_points[point_number] );
+            const array_1d<double, 3> normal = - GetGeometry().Normal( integration_points[point_number] );
```
Comment (RiccardoRossi) [inline applications/StructuralMechanicsApplication/custom_conditions/line_load_condition_2d.cpp]: ... why a minus here?

shouldn't it be the same convention as the 3d

**Comment by loumalouomega** [comment]
@RiccardoRossi Piiiiing

**Review by RiccardoRossi — APPROVED** [review]

File: applications/StructuralMechanicsApplication/custom_conditions/line_load_condition_2d.cpp (hunk 1 above)
Comment (loumalouomega) [reply applications/StructuralMechanicsApplication/custom_conditions/line_load_condition_2d.cpp]: I removed the minus

**Pull request merged**